setup.py

```diff
@@ -67,7 +67,7 @@ def package_vars(version_file):
     py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
     include_package_data=True,
     install_requires=[
-        "celery >= 4.3.0rc2",
+        "celery == 4.3.0",
```

Comment:
I think this is more accurately captures the version pinning we want to do:
```suggestion
        "celery >=4,<5",
```

The following quote is from [this source](https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/):

> It is not considered best practice to use `install_requires` to pin dependencies to specific versions, or to specify sub-dependencies (i.e. dependencies of your dependencies). This is overly-restrictive, and prevents the user from gaining the benefit of dependency upgrades.

Reply:
Fixed in e97e46a.

@king-alexander - note that you could have just clicked the "Commit suggestion" button above and created a suitable commit message.  That's why I did it that way.

Reply:
Thanks! Yes I saw but I wanted to include the versioning comment you suggested in the same commit.